persistence/upgrades/ravendb-5to6.md

```diff
@@ -165,6 +165,59 @@ In NServiceBus versions 3 and 4, RavenDB Persistence was part of the main NServi
 
 If attempting to convert directly from NServiceBus 4 or lower to NServiceBus 7 (which is _not recommended_) timeout data will have to be converted to the new format manually.
 
+### Patching legacy timeouts
+
+The following patch can be used to idempotently update all timeout documents to the new format.
```

Comment:
```suggestion
The following patch can be used to update all timeout documents to the new format.
```

While I can run the patch multiple times, I wasn't worried about it until you pointed it out. Not sure we need this.